pkg/util/utils.go

```diff
@@ -166,3 +168,26 @@ func FriendlyDuration(d time.Duration) time.Duration {
 	d2 := (d / time.Nanosecond) * (time.Nanosecond)
 	return d2
 }
+
+// CommandExecutesSuccessfully returns true if the command executed based on the exit code
+func CommandExecutesSuccessfully(cmd string, args ...string) bool {
+	var runner Runner = &RealRunner{}
+	var stdOut, stdErr io.Writer
+	// check in the path if cmd	is present
+	path, err := exec.LookPath(cmd)
```

Comment:
This is not needed. The underlying _exec.Command_ of the _Runner_ takes care of this.

Reply:
Okay

Reply:
See comment above ;-)